README.md

```diff
@@ -1,134 +1,122 @@
+When a proposal is first submitted, the Solid Editors will assign a single Editor as the shepherd for the proposal and assign a time window for review and its acceptance or rejection; the time window will be either 1 or 2 months from the date of proposal submission, depending on the complexity of the proposal. This time window includes all discussion and revision that may be needed to improve the proposal. The Solid Director, working with the Solid Editors and any administrative support personnel, will maintain a dashboard of open and closed proposals, time windows for review, and review status.
+
+If an Editor or other person disapproves of Candidate Proposal that is under consideration, they must provide substantive, written comments that contain proposed remedies to the Solid Editors at least 1 week before the end of the review time window. The Solid Editors will consider the comments and suggest possible remedies.
 
-Candidate Proposals without substantive changes require one Editor assigned to the material being modified to actively approve the proposal, with no Editors from the Editorial Team actively rejecting, and may be merged immediately by an assigned Editor. An assigned Editor may approve and merge their own non-substantive changes. Anyone from the Editorial Team has the right to revert a non-substantive change, but are encouraged to communicate with the assigned Editor that merged it first. Any revert must be accompanied by a reasonable explanation. If the change is reverted because they believe it to be substantive, that must be included in the explanation.
+Candidate Proposals with substantive changes must be responsive to comments and objections. If rough consensus cannot be reached among the Solid Editors, a vote may be taken during a meeting of the Editors, and the advancement of the proposal can be blocked by a negative vote by 2/3 of the Editors. Once the Candidate Proposal has successfully passed editorial review, it may be merged by an assigned Editor. The shepherd for a proposal is responsible for ensuring a resolution of the proposal within the time window, and if the shepherd is unable to complete the proposal review during the time window, the Solid Director will appoint a different Editor as shepherd and extend the time window for the proposal by 1 month.
```

Comment:
If a proposal is rejected/sent back for rework. Are there any limits on when it can be resubmitted?

Reply:
I made some tweaks to this paragraph to clarify; see what you think.